docs/docs/guides/enable-cloud-storage.mdx

```diff
@@ -0,0 +1,66 @@
+# Enable cloud storage on Lightdash
```

Comment:
@rephus is this just for self-hosted? if yes, can we update the title to "Enable.... (Self-Hosted)"

Reply:
it is mainly for self-hosted, because we have s3 enabled on our cloud

Reply:
cool, should we add `Self-Hosted` then in brackets? 

Or add a short description of how this is mostly for self-hosted (and in which cases you should do this if you're on cloud)?

Just so people know whether or not the doc applies to them :)